README.md

```diff
@@ -23,8 +23,14 @@ This package is bootstraped using [react-npm-boilerplate](https://github.com/jul
 
 ## Installation
 
+With Yarn:
 ```bash
-npm install react-spinners --save
+yarn add react-spinners
+```
+
+or alternatively using npm:
+```bash
+npm i -S react-spinners
```

Comment:
```suggestion
npm install react-spinners
```

I believe you don't need the save flag on newer versions of npm?

Reply:
I prefer to use --save flag...

Reply:
then let's do `npm install --save react-spinners`. I like to avoid abbreviations in documentations